### PR TITLE
Add PR labeler

### PR DESCRIPTION
**Pull request by core-man**
Related to #72

### CODE REVIEW

**Comment by core-man** [comment]
PR labelers are only added to software and appendix now.
Do we want to label maintenance although its rule will be complex?

**Comment by seisman** [comment]
> Do we want to label maintenance although its rule will be complex? At present, all the changes in `.github` will be labeled `maintenance`. We may add other globs.

I don't want to make it too complicated. The current rules look good to me.

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Add a labeler to any changes within folders or any subfolders
+
+# Seismological software: software name is used as the labeler
+cap:
+  - any: ['source/cap/**/*']
```
Comment (seisman) [inline .github/labeler.yml:5]: If I understand it correctly, the above line can be simplified to 
```
cap: source/cap/**/*
```

Perhaps we should change it to this more compact way?

Comment (core-man) [reply .github/labeler.yml]: done

**Review by seisman — APPROVED** [review]

**Pull request merged**